ModelBase/src/nodes/Reference.cpp

```diff
@@ -85,7 +85,8 @@ bool Reference::resolveHelper(bool indirect)
 
 	auto newTarget = computeTarget();
 
-	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
+	//TODO@cyril What is this for?
+//	Q_ASSERT(!newTarget || (newTarget->definesSymbol() && newTarget->symbolName() == name_));
```

Comment:
Yes, I checked and it is possible. So just declare your reference attribute like this:
ViewItemNode.h:

    ...
    ATTRIBUTE(Node, referencedNode, setReferencedNode)
    ...
ViewItemNode.cpp

    ...
    REGISTER_ATTRIBUTE(ViewItemNode, referencedNode, Node, false, false, true)
    ....